templates/embeds/collapsible.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends '@Tabler/embeds/card.html.twig' %}
+{% set collapsed = false %}
+{% set fullsize = true %}
```

Comment:
Parameter under `extends` works as untended?
I always put them above the extends , even more if the extend use it.

Reply:
To me an extends belong to the top of the file, to understand the structure.
But yes, it works even below. If you look at the compiled class for this template, you'll understand why.